docs/installation.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fa71c349-6c09-46cb-a3f2-8b0dcb67a0be",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "sidebar_position: 5\n",
+    "---\n",
+    "\n",
+    "# Installation\n",
+    "\n",
+    "<!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->\n",
+    "\n",
+    "## Basic Installation\n",
+    "\n",
+    "A basic installer is packaged as a container image and can be run directly. This installation method is only intended to work in a basic cloud project free of any significant constraints (i.e. GCP Organizational Policies, etc). This path has been tested on brand new cloud projects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eba1f962-b486-485b-abc4-3a1bc7c9dce5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!docker run -it \\\n",
+    "  -v $HOME/.kube:/root/.kube \\\n",
+    "  -e PROJECT=$(gcloud config get project) \\\n",
+    "  -e TOKEN=$(gcloud auth print-access-token) \\\n",
+    "  substratusai/installer gcp-up.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "866b8960-afab-4506-9a16-2f0662ea203a",
+   "metadata": {},
+   "source": [
+    "## Advanced Installation\n",
+    "\n",
+    "Most enterprises will need to modify the base configuration files to comply with constraints specific to their environment. The Terraform and Kubernetes configurations do not attempt to export every option as a variable. In order to keep the configurations simple, most options are set directly in the `.tf` and `.yaml` files.\n",
+    "\n",
+    "To set advanced installation options, first clone the substratus repo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7aa46d59-0439-47ff-a104-812607d0b71f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/substratusai/substratus && cd ./substratus"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdde4876-4821-4ff8-a74e-6304a23af98f",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir('./substratus')"
```

Comment:
Ahh and you ran into the issue here. Use the magic command above and this can go